### ISSUE TITLE
IndexOutOfBoundsException

### ISSUE THREAD

**Issue by neeraj-goodbits** [Appboy/appboy-android-sdk#156]
Firebase reported a crash in our project.
The log is given below.

We use the SDK Version : 6.0.0
**com.appboy:android-sdk-ui-x:6.0.0**

```
Caused by java.lang.IndexOutOfBoundsException: Index: 1, Size: 1
       at java.util.ArrayList.get(ArrayList.java:437)
       at com.appboy.ui.contentcards.AppboyCardAdapter.markOnScreenCardsAsRead(AppboyCardAdapter.java:175)
       at com.appboy.ui.AppboyContentCardsFragment.onPause(AppboyContentCardsFragment.java:125)
       at androidx.fragment.app.Fragment.performPause(Fragment.java:2879)
       at androidx.fragment.app.FragmentStateManager.pause(FragmentStateManager.java:373)
       at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1204)
       at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1354)
       at androidx.fragment.app.FragmentManager.moveFragmentToExpectedState(FragmentManager.java:1432)
       at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1495)
       at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:2617)
       at androidx.fragment.app.FragmentManager.dispatchPause(FragmentManager.java:2585)
       at androidx.fragment.app.Fragment.performPause(Fragment.java:2872)
       at androidx.fragment.app.FragmentStateManager.pause(FragmentStateManager.java:373)
       at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1204)
       at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1354)
       at androidx.fragment.app.FragmentManager.moveFragmentToExpectedState(FragmentManager.java:1432)
       at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1495)
       at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:2617)
       at androidx.fragment.app.FragmentManager.dispatchPause(FragmentManager.java:2585)
       at androidx.fragment.app.Fragment.performPause(Fragment.java:2872)
       at androidx.fragment.app.FragmentStateManager.pause(FragmentStateManager.java:373)
       at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1204)
       at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1354)
       at androidx.fragment.app.FragmentManager.moveFragmentToExpectedState(FragmentManager.java:1432)
       at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1495)
       at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:2617)
       at androidx.fragment.app.FragmentManager.dispatchPause(FragmentManager.java:2585)
       at androidx.fragment.app.FragmentController.dispatchPause(FragmentController.java:280)
       at androidx.fragment.app.FragmentActivity.onPause(FragmentActivity.java:419)
```
      
Please look into this and get it sorted out.

**Comment by radixdev** [comment]
Thank you for the issue. I've added this as a high priority bug and a fix should be available in the next release

**Comment by radixdev** [comment]
Thank you for your patience!

There is a fix available in version 7.0.0 of SDK, available today
